docs/examples/promise/12_chose_fee_asset_to_pay_tx_fee/index.js

```diff
@@ -8,17 +8,12 @@ const { Keyring } = require('@polkadot/keyring');
 // utility function for random values
 const {randomAsU8a} = require('@cennznet/util');
 
-// some constants we are using in this sample
-const ALICE = '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY';
-
-// BOB is pool liquidity provider
-const BOB = '5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty';
 const AMOUNT = 10000;
 const FEE_ASSET_ID = 16000;
 const MIN_REQUIRED_POOL_BALANCE = 1000000;
 
 // Asset Id for CENNZ in Rimu
```

Comment:
Perhaps we need to change Rimu to testnet or Nikau

Reply:
I am going thro' all the examples to fix the testnet to Nikau and check if any other comments needs to any modification